### ISSUE TITLE
Using -noconsole on Windows 7 prevents DOSBox from starting.

### ISSUE THREAD

**Issue by jpetriejr** [dosbox-staging/dosbox-staging#1775]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### What Operating System are you using?

Windows 7

### Relevant hardware info

Intel Core2 Duo 3 GHZ

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

Passing -noconsole as a command line parameter prevents DOSBox from starting on Windows 7.  The console will get to "Loguru initialized" and just quit.

### Steps to reproduce the behaviour.

On a Windows 7 PC do the following:
1.  Open a command prompt window where dosbox.exe is located.
2.  Run dosbox by using dosbox -noconsole

It will run and immediately quit.

### Your configuration

```ini
Just a stock config.
```

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by kcgen** [comment]
Thanks @jpetriejr .
Interesting; anyone on Win 10/11 want to try this too?

**Comment by jpetriejr** [comment]
It works fine on Windows 10.  Should have mentioned that.

**Comment by jpetriejr** [comment]
@kcgen Although, on Windows 10 (not sure about 11) if you do pass -noconsole it briefly displays then closes which is kind of not ideal and mainline DOSBox does not do this.

**Comment by jpetriejr** [comment]
@kcgen  I just figured out what was causing this issue and I don't think it is necessarily related to Windows 7 or DOSBox.  It's a permission issue with writing stderr.txt and stdout.txt to the folder the EXE is in.  On the computer I was having this issue, I purposefully have it setup where my user is just a regular user for testing stuff like this.  In any case I gave my user full rights to the folder and was able to start of DOSBox without the console.  

Here is where it gets weird.  In this scenario I'm launching DOSBox from my own application and it writes the output text files (or attempts to do so).  Running a regular installed DOSBox with -noconsole doesn't attempt to write the files it seems as it runs ok.  This whole thing appears to be my problem.  I'm using CreateProcessA() to start up the process so time to dig into how to prevent this.  I have a feeling I will have to intercept the outputs and prevent them from writing to files.
